backend/app/api/endpoints/adapter/tasks.py

```
@@ -359,9 +363,17 @@ async def export_task_docx(
     if not task:
         raise HTTPException(status_code=404, detail="Task not found")
 
+    # Parse message_ids if provided
+    filter_message_ids: Optional[list[int]] = None
+    if message_ids:
+        try:
+            filter_message_ids = [int(id.strip()) for id in message_ids.split(",") if id.strip()]
+        except ValueError:
+            raise HTTPException(status_code=400, detail="Invalid message_ids format. Must be comma-separated integers.")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)